src/gcinfo/CMakeLists.txt

```diff
@@ -23,5 +23,15 @@ endif(CLR_CMAKE_PLATFORM_UNIX)
 add_subdirectory(lib)
 add_subdirectory(crossgen)
 
+if (CLR_CMAKE_PLATFORM_ARCH_I386)
+    # On x86, build RyuJIT/ARM32 cross-compiling altjit.
+    add_subdirectory(arm_altjit)
```

Comment:
Because these are builds of the `gcinfo` library, and the `gcinfo` library is logically distinct from the JIT.